src/MaterialEditor.Core/Core.MaterialEditor.cs

```diff
@@ -886,10 +887,11 @@ private static Shader LoadShader(string shaderName, string assetBundlePath, stri
 
 #if KK || EC || KKS
         public override bool CheckBlacklist(string materialName, string propertyName)
-        {
-            if (materialName == "cf_m_body" || materialName == "cm_m_body")
-                if (propertyName == "alpha_a" || propertyName == "alpha_b" || propertyName == "AlphaMask")
-                    return true;
+        {
+            if (propertyName == "alpha_a" || propertyName == "alpha_b") return true;
+
+            if (Regex.IsMatch(materialName, @"^c[mf]_m_body(\.MECopy\d+)?$") && propertyName == "AlphaMask") return true;

```

Comment:
Actually, can you use this constant just in case? https://github.com/IllusionMods/KK_Plugins/blob/d79e22073b31a9dac87964138386c2b2590309bd/src/MaterialEditor.Base/MaterialAPI.cs#L21